lib/evss/disability_compensation_form/data_translation.rb

```diff
@@ -154,24 +154,25 @@ def get_banking_info
       end
 
       def translate_mailing_address(address)
-        pciu_address = { 'country' => address['country'],
-                         'addressLine1' => address['addressLine1'],
-                         'addressLine2' => address['addressLine2'],
-                         'addressLine3' => address['addressLine3'],
-                         'effectiveDate' => address['effectiveDate'] }
+        pciu_address = { 'effectiveDate' => address['effectiveDate'], 'country' => address['country'],
```

Comment:
is the only thing changing here the order/formatting?

Reply:
ya, otherwise rubocop yells at me